app/lib/actions/katello/activation_key/update.rb

```diff
@@ -15,15 +15,27 @@ module Katello
     module ActivationKey
       class Update < Actions::EntryAction
         def plan(activation_key, activation_key_params)
+          cp_call = cp_changed?(activation_key.auto_attach, activation_key_params[:auto_attach]) ||
+                    cp_changed?(activation_key.service_level, activation_key_params[:service_level]) ||
+                    cp_changed?(activation_key.release_version, activation_key_params[:release_version])
```

Comment:
I might wrap this in a function call itself as well, so that further down you could just do:

```
if update_candlepin?(activation_key, params)
....
...
end
```

This encapsulates the logic and allows anyone reading the code to skip over this chunk if they don't need to see the details of it.